services/apps/data_sink_worker/src/service/organization.service.ts

```diff
@@ -90,6 +89,11 @@ export class OrganizationService extends LoggerBase {
             'founded',
             'attributes',
           ]
+
+          if (!existing.displayName || data.displayName.trim().length === 0) {
```

Comment:
In the second clause you sure it’s `data` and not `existing`?

Reply:
good catch - was too quick there :/